### DISCUSSION TITLE
v1.2.0-alpha

### DISCUSSION THREAD

**Discussion by warrensbox** [warrensbox/terraform-switcher#459] [Releases]
## What's Changed
* 154 feature request option to dry run by @MatrixCrawler in https://github.com/warrensbox/terraform-switcher/pull/416
* feat(goreleaser): Enable Release Changelog by @yermulnik in https://github.com/warrensbox/terraform-switcher/pull/418
* docs: Streamline/fix docu site files and README by @yermulnik in https://github.com/warrensbox/terraform-switcher/pull/419
* go: bump golang.org/x/sys from 0.19.0 to 0.20.0 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/426
* go: bump golang.org/x/crypto from 0.22.0 to 0.23.0 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/428
* docs: Fix links to `install.sh` by @yermulnik in https://github.com/warrensbox/terraform-switcher/pull/432
* fix: Only extract terraform file from Terraform release archive by @MatthewJohn in https://github.com/warrensbox/terraform-switcher/pull/433
* go: bump github.com/hashicorp/go-version from 1.6.0 to 1.7.0 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/438
* go: bump github.com/gookit/slog from 0.5.5 to 0.5.6 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/425
* go: bump github.com/spf13/viper from 1.18.2 to 1.19.0 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/445
* go: bump github.com/gookit/slog from 0.5.5 to 0.5.6 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/444
* RECENT to JSON Proposal by @MatrixCrawler in https://github.com/warrensbox/terraform-switcher/pull/437
* Update README.md by @MatrixCrawler in https://github.com/warrensbox/terraform-switcher/pull/448
* skip any parsing of config files if the -v flag is provided by @MatrixCrawler in https://github.com/warrensbox/terraform-switcher/pull/446
* Added ERRORlog-level by @MatrixCrawler in https://github.com/warrensbox/terraform-switcher/pull/447
* Fix for #443 - Error unzipping on windows systems by @MatrixCrawler in https://github.com/warrensbox/terraform-switcher/pull/450
* gh-actions: bump goreleaser/goreleaser-action from 5 to 6 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/453
* go: bump golang.org/x/sys from 0.20.0 to 0.21.0 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/454
* go: bump golang.org/x/crypto from 0.23.0 to 0.24.0 by @dependabot in https://github.com/warrensbox/terraform-switcher/pull/455
* OpenTofu support and JSON recentFile support by @MatthewJohn in https://github.com/warrensbox/terraform-switcher/pull/435
* Check version sources based on whether verison has been set by @MatthewJohn in https://github.com/warrensbox/terraform-switcher/pull/429

**Full Changelog**: https://github.com/warrensbox/terraform-switcher/compare/v1.1.1...v1.2.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/warrensbox/terraform-switcher/releases/tag/v1.2.0-alpha'>v1.2.0-alpha</a>.</em>

**Comment by yermulnik** [comment]
JFYI: I'll create a PR to update [CHANGELOG](https://github.com/warrensbox/terraform-switcher/blob/master/CHANGELOG.md) for this alpha release tomorrow (will do my best to align with https://github.com/warrensbox/terraform-switcher/releases/tag/v1.2.0-alpha release changelog)

**Reply by warrensbox** [reply]
Thanks. Github automatically created this release doc for me. I only had to push a button. It's a nice feature.

**Reply by yermulnik** [reply]
https://github.com/warrensbox/terraform-switcher/pull/460
